Automattic/jetpack
# [Plugin] Backup: Handle period before first backup starts

#### Changes proposed in this Pull Request:
When a plan is first purchased the jetpack-backup page is loaded before the first backup can start.  Without this fix the user receives an empty grey bar.  This PR adds some text that the backup will start soon and starts polling the end point to detect the first backup.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Create a jurassic ninja site and install this branch of the backup plugin
* Purchase a plan normally
* From checkout you should redirect back to the backup plugin
* Ensure the screen is not blank, and loads into the progress screen after the backup starts
